app/services/search/user.rb

```diff
@@ -29,7 +29,8 @@ def username_query(username)
               analyze_wildcard: true,
               allow_leading_wildcard: false
             }
-          }
+          },
+          size: Search::Postgres::Username::MAX_RESULTS # Limit the number of results we return to the frontend
```

Comment:
My thinking here is that since Elasticsearch is going away anyways that it's okay for the constant to live in the `Search::Postgres` namespace vs one that's a little more "broad".